clang/docs/HLSL/HLSLSupport.rst

```diff
@@ -114,6 +114,54 @@ not re-targetable, we want to share the Clang CodeGen implementation for HLSL
 with other GPU graphics targets like SPIR-V and possibly other GPU and even CPU
 targets.
 
+hlsl.h
+------
+
+HLSL has a library of standalone functions. This is similar to OpenCL and CUDA,
+and is analogous to C's standard library. The implementation approach for the
+HLSL library functionality draws from patterns in use by OpenCL and other Clang
+resource headers. All of the clang resource headers are part of the
+``ClangHeaders`` component found in the source tree under
+`clang/lib/Headers <https://github.com/llvm/llvm-project/tree/main/clang/lib/Headers>`_.
+
+.. note::
```

Comment:
Comment for reviewers: GitHub's preview doesn't properly render the `note` block. An example of this in HTML can be found here: https://clang.llvm.org/docs/LanguageExtensions.html#has-builtin